src/character.h

```diff
@@ -1833,6 +1833,7 @@ class Character : public Creature, public visitable
 
         // --------------- Proficiency Stuff ----------------
         bool has_proficiency( const proficiency_id &prof ) const;
+        float get_proficiency_level( const proficiency_id &prof ) const;
```

Comment:
Could you change this name from `_level` to `_practice` or similar? `level` has connotations that I don't think are quite correct here.

Reply:
Sure! Done.